src/View/GetHelp.elm

```diff
@@ -64,16 +76,20 @@ columnStyle =
         , flexWrap wrap
         , flexDirection column
         , justifyContent spaceBetween
-        , withMedia [ only screen [ Media.minWidth (px 769) ] ]
+        , withMediaTablet
             [ alignItems flexStart
             , flexDirection row
             , flexWrap noWrap
             ]
         ]
 
 
-card : String -> String -> String -> CallToAction -> Html msg
-card title quote description call =
+
+-- This function is a little bit eek!
```

Comment:
I'm sad about this - but not taking action (yet)

Reply:
If you don't want to pass the viewportWidth an alternative might be to  have renderCallToAction return
```
p [ css [ display none, withMediaMobile display block ] ] [ ... ] 
p [ css [ display block, withMediaMobile display none ] ] [ ... ] 
```

Reply:
Neat!

It's more that the function has so many arguments which is usually a bad sign & leads to broken-ness. We don't need to refactor now though.